### DISCUSSION TITLE
Connection refused?

### DISCUSSION THREAD

**Discussion by HyptexPvP** [NginxProxyManager/nginx-proxy-manager#3270] [Q&A] [answered]
Im setting up pterodactyl + nginx proxy manager but im running into some issues.

i setup a proxy host   
![ProxyNoSSL](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/63811460/9342ba1b-26cd-464d-bf30-a6170ef8f6d5)
it gives me a ERR_SSL_UNRECOGNIZED_NAME_ALERT error. so i added a ssl cert! it gave me a 502 bad gateway.
but if i do curl 0.0.0.0:8000 i can see the html, also if i do serverIP:8000 i can see it too.
i went to the logs and i see 

` 2023/10/19 02:07:57 [error] 400#400: *544 connect() failed (111: Connection refused) while connecting to upstream, client: MY IP, server: panel.domain.tld, request: "GET /favicon.ico HTTP/2.0", upstream: "http://0.0.0.0:8000/favicon.ico", host: "panel.domain.tld", referrer: "https://panel.domain.tld/"`

my docker-compose for nginx proxy manager
![proxy](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/63811460/4409cfd7-86a3-448c-85d9-0c17c859e82c)

my docker-compose for pterodactyl: https://pastebin.com/c6qjiyb9

**Comment by crocofied** [comment]
Got the same error

**Comment by glkuhlmann** [comment]
I know this question is old but did you ever find the solution?  I am running into the exact same issue.

**Reply by HyptexPvP** [reply, marked as answer]
Yes, just use docker DNS instead so if your other container name is panel the hostname would be panel
